### PR TITLE
Add detail and demo to README

### PR DESCRIPTION
**Pull request by Xcodo**
This is a resolution to issue #109 and also adds a demo GIF (including the right-click menu) in place of the static screenshot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ Shows the rendered HTML of the AsciiDoc content in the current editor.
 
 You can install this module from the command-line by typing `apm install asciidoc-preview`.
 You can toggle the preview pane, which appears to the right of the editor, using the key combination `ctrl-shift-a`.
+More options are accessed in by right-clicking on the preview.
```
Comment (mojavelinux) [inline README.md]: Can you change "accessed in" to "accessible" ?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ Shows the rendered HTML of the AsciiDoc content in the current editor.
...
 
 Thanks to @kevinsawicki! I just adapted [markdown-preview](https://github.com/atom/markdown-preview) and used [asciidoctorjs-npm-wrapper](https://github.com/anthonny/asciidoctorjs-npm-wrapper).
 
-![](https://raw.githubusercontent.com/anthonny/asciidoc-preview/master/screenshot.jpg)
+![AsciiDoc Preview demo](https://cloud.githubusercontent.com/assets/6715247/11607696/0f555d54-9b49-11e5-8c72-31b718289562.gif)
```
Comment (mojavelinux) [inline README.md]: Could you add the gif to the pull request instead of linking to an attachment?

Comment (Xcodo) [inline README.md]: Sure. The previous version linked to an absolute URL in a different (the original?) repo. The docs suggest that a relative link should work though.

**Comment by mojavelinux** [comment]
I think the absolute path is used so that it shows up in the Atom package repo.

**Comment by Xcodo** [comment]
Done those (fail on proof reading!)

The original recording was 749 kB and I took down the resolution. The original could go in instead if you prefer it:
Imgur
http://imgur.com/2gleCb4

**Comment by Xcodo** [comment]
It would probably make sense to change the URL to the asciidoctor/atom-asciidoc-preview repo afterwards too.

**Comment by mojavelinux** [comment]
Yes, please change the URL as part of the pull request. Then, it looks good to merge. @anthonny?

**Comment by anthonny** [comment]
:+1: thank you @Xcodo :beers:

**Pull request merged**